app/code/Magento/Customer/composer.json

```diff
@@ -25,7 +25,8 @@
         "magento/module-tax": "*",
         "magento/module-theme": "*",
         "magento/module-ui": "*",
-        "magento/module-wishlist": "*"
+        "magento/module-wishlist": "*",
+        "magento/module-persistent": "100.2.*"
```

Comment:
What is that for?

Reply:
Hi @orlangur,

This is my mistake in code commit, I will squash the rest of commits.